csrc/fusion_executor/executor_utils.cpp

```diff
@@ -299,7 +299,8 @@ std::unique_ptr<caching::VectorizedTensorInfo> getVectorizedTensorValidationInfo
 
     auto vector_dim = vector_info.vectorized_loop_id;
     const auto is_aligned =
-        vector_dim->getParallelType() == ParallelType::Vectorize;
+        vector_dim->getParallelType() == ParallelType::Vectorize ||
+        vector_dim->getParallelType() == ParallelType::Group;
```

Comment:
Why is this change necessary?

Reply:
Because grouped reduction tv is added to `vectorized_set_info`,  if not aligned, must be fusion input or output.
If don't directly using `VectorizeValidator::validate(tv)`, won't be added to `vectorized_set_info` and this change is not required.